styles/navbar.css

```diff
@@ -0,0 +1,18 @@
+.header{
```

Comment:
Reminder to work on the mobile version. Remember the initial styling is for mobile and media queries are for desktop or tablets (if you desire)